emr/bootstrap-pyspark

```diff
@@ -104,7 +104,7 @@ pyarrow \
 s3fs \
 boto3 \
 pandas \
-pytz \
+tzdata \
```

Comment:
Is this a Windows-only requirement? I could be wrong but I thought I read that Linux would already have the time database.

Reply:
I guess so?

Reply:
I don't think it will be needed for this EMR Linux image.

It is generally recommended that any time zone libraries should attempt to use the system data before using the tzdata package, but some systems (notably Windows) do not deploy zoneinfo binaries of this type, and so tzdata is necessary.
https://tzdata.readthedocs.io/en/latest/#overview